bluetooth: fast_pair: use_case: add input_device use case (#16601)
nrfconnect/sdk-nrf: doc/nrf/external_comp/bt_fast_pair.rst

You can use the :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_UNKNOWN` Kconfig option to implement use cases that are not yet supported by the |NCS|.
In this case, you must manually enable the required Fast Pair features and extensions in the application's Kconfig configuration.

.. _ug_bt_fast_pair_use_case_input_device:

Input device
============

> Input device is a Human Interface Device (HID) such as a mouse, keyboard, remote control or a gaming pad that lets the user interact with electronic devices such as a PC, TV or a console.

Reviewer: ```suggestion
Input device is a Human Interface Device (HID) such as a mouse, keyboard, remote control, or gaming pad used to interact with electronic devices such as a PC, TV, or console.
```

> If your product is targeting the input device use case, you must enable the :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_INPUT_DEVICE` Kconfig option that automatically selects the appropriate Fast Pair configuration for it.

Reviewer: ```suggestion
If your product is targeting the input device use case, you must enable the :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_INPUT_DEVICE` Kconfig option that automatically selects the appropriate Fast Pair configuration.
```

> As the input device feature requirements are not yet defined in the Google Fast Pair specification, the selected Fast Pair feature and extension set for this use case is chosen arbitrarily to provide the best user experience.

Reviewer: ```suggestion
Currently, the Google Fast Pair specification does not define the input device feature requirements.
For this reason, the Fast Pair feature and extension set is chosen arbitrarily to ensure the best user experience.
```

> You should declare support for the input device use case during the device registration process in the Google Nearby Device console.

Reviewer: ```suggestion
You must declare support for the input device use case when registering your device in the Google Nearby Device console.
```

> To activate the support, select :guilabel:`Input Device` option in the **Device Type** list of the **Fast Pair** protocol configuration panel.

Reviewer: ```suggestion
To enable the support, select the :guilabel:`Input Device` option from the **Device Type** list in the **Fast Pair** protocol configuration panel.
```

   It is recommended to use the special mode of the ``Fast Pair Procedure`` for the locator tag use case (see :ref:`ug_bt_fast_pair_gatt_service_no_ble_pairing` for more details).
   The Bluetooth bonding information can cause connection establishment issues and delays on some Android devices.

.. _ug_bt_fast_pair_use_case_mouse:

Mouse
=====

> Mouse is a Human Interface Device (HID) that lets the user interact with electronic devices such as a PC.

Reviewer: ```suggestion
Mouse is a Human Interface Device (HID) used to interact with electronic devices such as a PC.
```

> This use case variant is a more specific version of the :ref:`ug_bt_fast_pair_use_case_input_device` use case and shares a lot of similarities with it.

Reviewer: ```suggestion
This use case is a specific variant of the :ref:`ug_bt_fast_pair_use_case_input_device` and shares many similarities with it.
```
simplified a bit

> If your product is targeting the mouse use case, you must enable the :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_MOUSE` Kconfig option that automatically selects the appropriate Fast Pair configuration for it.

Reviewer: ```suggestion
If your product is targeting the mouse use case, you must enable the :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_MOUSE` Kconfig option that automatically selects the appropriate Fast Pair configuration.
```

> As the mouse feature requirements are not yet defined in the Google Fast Pair specification, the selected Fast Pair feature and extension set for this use case is chosen arbitrarily to provide the best user experience.
> 
> You should declare support for the mouse use case during the device registration process in the Google Nearby Device console.
> To activate the support, select :guilabel:`Mouse` option in the **Device Type** list of the **Fast Pair** protocol configuration panel.

Reviewer: ```suggestion
Currently, the Google Fast Pair specification does not specify the mouse feature requirements. 
For this reason, the Fast Pair feature and extension set is chosen arbitrarily to ensure the best user experience.

You must declare support for the mouse use case when registering your device in the Google Nearby Device console.
To enable the support, select the :guilabel:`Mouse` option from the **Device Type** list in the **Fast Pair** protocol configuration panel.
```
